src/palace/manager/celery/importer.py

```diff
@@ -0,0 +1,53 @@
+from celery.canvas import Signature
+from sqlalchemy.orm import Session
+from sqlalchemy.sql import Select
+
+from palace.manager.service.redis.models.lock import RedisLock
+from palace.manager.service.redis.redis import Redis
+from palace.manager.sqlalchemy.model.collection import Collection
+from palace.manager.util.log import LoggerType, pluralize
+
+
+def import_all(
```

Comment:
I created some general helper functions here that I think can be shared and used by all the various collection import tasks we will end up adding to Celery in the next few sprints.